ui/app/appointments/services/checkinPopUp.js

```diff
@@ -16,8 +16,8 @@ angular.module('bahmni.common.uiHelper')
                 className: config.className || 'ngdialog-theme-default'
             });
 
-            scope.performAction = function (close, time) {
-                scope.action(time, close);
+            scope.performAction = function (time, callbackFn) {
```

Comment:
time is assigned to scope but still being passed  from UI i.e. html
time as well as close should not be passed from UI. Please check if there is a better way to do it.